src/onboarding/style.ts

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components/native';
+import { Button, Typography, Checkbox } from '../_components';
+
+export const ViewContainer = styled.ScrollView`
+  background-color: ${({ theme }) => theme.colors.white};
+`;
```

Comment:
I had to do this for the `DiagonalSplitView` as well.. Maybe we can create a separate `Layout` component where these defaults live? We could add the `contentContainerStyle={{ flexGrow: 1 }}` there and also a `SafeAreaView`.

Reply:
Added a SafeAreView component to cover this, implementation is up for debate